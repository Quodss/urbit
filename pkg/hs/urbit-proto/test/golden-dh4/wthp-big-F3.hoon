%.  [& & & & & & & & & & & & & & & & & & & & & & & & & & & & & & & & & & &]
|=  big=[? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ? ?]
^-  @
?-  big
  [_ _ _ _ 0 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _]  1
  [_ _ _ _ _ 1 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _]  2
  [_ _ _ _ _ _ _ _ _ _ _ _ _ _ 0 _ 0 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _]  3
  [_ _ _ _ _ _ _ _ _ _ _ _ _ _ 1 _ 0 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _]  4
  [_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 0 _ _ _ _ _ _ _ _ _ _ _ _ _ 0 _ _ _ _]  5
::  [_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 1 _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _]  6
==
