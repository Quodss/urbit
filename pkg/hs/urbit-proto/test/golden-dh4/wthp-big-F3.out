PARSED

%.  :*  %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
        %&
    ==
|=  ^=  big
    :*  ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
        ?
    ==
^-  @
?-  big
    :*  _
        _
        _
        _
        0
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
    ==
  1
::
    :*  _
        _
        _
        _
        _
        1
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
    ==
  2
::
    :*  _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        0
        _
        0
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
    ==
  3
::
    :*  _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        1
        _
        0
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
    ==
  4
::
    :*  _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        0
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        _
        0
        _
        _
        _
        _
    ==
  5
::
==

SOFT

%:  |=  ^=  big
        :*  ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
        ==
    ^-  @
    ?:  ?=  :*  _
                _
                _
                _
                %0
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
            ==
        big
      1
    ?:  ?=  :*  _
                _
                _
                _
                _
                %1
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
            ==
        big
      2
    ?:  ?=  :*  _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                %0
                _
                %0
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
            ==
        big
      3
    ?:  ?=  :*  _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                %1
                _
                %0
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
            ==
        big
      4
    ??  ?=  :*  _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                %0
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                _
                %0
                _
                _
                _
                _
            ==
        big
    5
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
    %&
==

HARD

=+  =:  ^=  big
        :*  ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
            ?
        ==
      ^-  *
      0
    |%
    ++  %
      ^-  @
      ?:  ?=  :*  _
                  _
                  _
                  _
                  %0
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
              ==
          big
        1
      ?:  ?=  :*  _
                  _
                  _
                  _
                  _
                  %1
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
              ==
          big
        2
      ?:  ?=  :*  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  %0
                  _
                  %0
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
              ==
          big
        3
      ?:  ?=  :*  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  %1
                  _
                  %0
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
              ==
          big
        4
      ??  ?=  :*  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  %0
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  _
                  %0
                  _
                  _
                  _
                  _
              ==
          big
      5
    --
%=  %.+2
    +6
  =>  +3
  :*  %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
      %&
  ==
::
==

ERROR

  trace:  play:
          lvl   0
          sut   0|*
          ----
          code  %:  |=  ^=  big
                        :*  ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                            ?
                        ==
                    ^-  @
                    ?:  ?=  :*  _
                                _
                                _
                                _
                                %0
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                            ==
                        big
                      1
                    ?:  ?=  :*  _
                                _
                                _
                                _
                                _
                                %1
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                            ==
                        big
                      2
                    ?:  ?=  :*  _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                %0
                                _
                                %0
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                            ==
                        big
                      3
                    ?:  ?=  :*  _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                %1
                                _
                                %0
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                            ==
                        big
                      4
                    ??  ?=  :*  _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                %0
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                _
                                %0
                                _
                                _
                                _
                                _
                            ==
                        big
                    5
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                    %&
                ==


          play:
          lvl   0
          sut   0|*
          ----
          code  |=  ^=  big
                    :*  ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                    ==
                ^-  @
                ?:  ?=  :*  _
                            _
                            _
                            _
                            %0
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                        ==
                    big
                  1
                ?:  ?=  :*  _
                            _
                            _
                            _
                            _
                            %1
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                        ==
                    big
                  2
                ?:  ?=  :*  _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            %0
                            _
                            %0
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                        ==
                    big
                  3
                ?:  ?=  :*  _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            %1
                            _
                            %0
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                        ==
                    big
                  4
                ??  ?=  :*  _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            %0
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            %0
                            _
                            _
                            _
                            _
                        ==
                    big
                5


          tire:
          fish
                >>  :-  :*  _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            %0
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            _
                            %0
                            _
                        ==
                    _

                >>  :-  [_ _ _ _ _ _ _ _ _ _ _ _ _ _ %0 _ %0 _]
                    _

                >>  :-  [_ _ _ _ _ _ _ _ _ _ _ _ _ _ %1 _ %0 _]
                    _

                >>  [[_ _ _ _ _ %1 _] _]
                >>  [[_ _ _ _ %0 _] _]


          type  :-  ^=  big
                    :*  ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                        ?
                    ==
                0|*



  tire-fish:
  axis  2
  miss  :-  :*  _
                _
                _
                _
                %1
                %0
                _
                _
                _
                _
                _
                _
                _
                _
                %0
                _
                %1
                _
            ==
        _


